content/departments/cloud/technical-docs/managed-smtp/index.md

```diff
@@ -0,0 +1,89 @@
+
+> NOTE: **Managed SMTP is currently an opt-in feature**, and is undergoing a gradual rollout. For more details, refer to [RFC 705](https://docs.google.com/document/d/1eaShaXlpMEezawuwTZ26nuo5g_1MjQmjpp8VvQTokzw/edit) and [the tracking issue](https://github.com/sourcegraph/customer/issues/1408). The goal is to eventually provide managed SMTP to all instances by default.
+
+Managed SMTP allows Sourcegraph Cloud instances to have email delivery services configured out-of-the-box.
+SMTP services are currently backed by [SparkPost EU](https://app.eu.sparkpost.com).
+
+In this document:
+
+- [Enabling managed SMTP for a Cloud instance](#enabling-managed-smtp-for-a-cloud-instance) - links to guides
+- [Vendor management](#vendor-management) - management details for the SMTP vendor
+- [Monitoring](#monitoring) - monitoring SMTP capabilities
+- [Vendor integration](#vendor-integration) - high-level overview of how the integration with the vendor works
+
+## Enabling managed SMTP for a Cloud instance
+
+Managed SMTP is currently only available for Cloud v1.1 - see [the guide here](../v1.1/mi1-1_managed_smtp.md).
+
+## Vendor management
+
+- **Account**:
+  - Okta: WIP
+  - Root account credentials: [Cloud Team - SparkPost](https://start.1password.com/open/i?a=HEDEDSLHPBFGRBTKAKJWE23XX4&h=team-sourcegraph.1password.com&i=k5oim4jlnqnuqsqsjrs344nhsi&v=qxzajcksgc3givogl3r6qjbimu) (note that you must log in via [SparkPost EU](https://app.eu.sparkpost.com))
+- **Billing**: [Airbase Virtual Corporate Card](https://dashboard.airbase.io/services/323464)
+  - We are billed based on emails delivered according to our usage plan, included currently up to 250,000 (as of Dec 2022), after which we are billed for overages.
+- **API Keys**: [list](https://app.eu.sparkpost.com/account/api-keys) - see [vendor integration](#vendor-integration) for more details
+
+## Monitoring
+
+### Application-side
+
+- Alerting: [frontend: email_delivery_failures](https://docs.sourcegraph.com/admin/observability/alerts#frontend-email-delivery-failures)
+- Dashboards: [Frontend: Email delivery](https://docs.sourcegraph.com/admin/observability/dashboards#frontend-email-delivery)
+- [Multi-instance dashboard](../observability/index.md#multi-instance-dashboard): [Frontend: Total emails successfully delivered every 5 minutes](https://monitoring.sgdev.org/d/multi-instance-overviews/multi-instance-overviews?orgId=1)
+
+### Vendor-side
+
+Dashboards are available [in the SparkPost web application](https://app.eu.sparkpost.com/signals/analytics?range=day&timezone=America/Vancouver&precision=hour&metrics%5B0%5D=count_targeted&metrics%5B1%5D=count_rendered&metrics%5B2%5D=count_accepted&metrics%5B3%5D=count_bounce&report=summary). Relevant metrics to look at:
+
+- Accepted count: we are billed for this based on our usage plans, up to 250,000 (as of Dec 2022).
+- Bounce rates: this indicates emails that are being rejected by recipients due to misconfiguration (invalid recipients) or spam filters.
```

Comment:
do we have alerting for increasing bounce rates?

Reply:
No, see below under "Future work" - in rollout `mi check` will periodically give bounce rate analytics (once a month as part of instance upgrade test plan). We will need a cron service to poll deliverability statistics, or a service to ingest webhook events, in order to do alerting

Reply:
Follow-up: https://github.com/sourcegraph/customer/issues/1662